### DISCUSSION TITLE
Logged 2 times the same update while on service

### DISCUSSION THREAD

**Discussion by djpalmis** [teslamate-org/teslamate#3778] [Q&A] [answered]
Hi, is it normal that while in service it logged a firmware update?
Even if the firmware was the same? Any idea about what happened?
![Screenshot_20240325_195107_Samsung Internet](https://github.com/teslamate-org/teslamate/assets/402122/e53e4fdb-f12f-4a6b-9781-ef56cce16ff3)

**Comment by cwanja** [comment, marked as answer]
Did service install the same update twice? 

Simply put - it logs what happens from the API. I have had a firmware reinstalled five times in service. Depends on what they replace and if they have to flash the firmware.

**Reply by djpalmis** [reply]
They changed headlights, apparently they needed to update the firmware.
Thanks for the reply!
